1619625075602 [Opening serial port: /dev/ttyTHS1.] 
1619625075903 [The robot is connected to ] [/dev/ttyTHS1] 
1619625075903 [Initial robot's topic...] 
1619625083745 [Data from: [ ]] 
1619625083746 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083751 [Data from: [w]] 
1619625083752 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083783 [Data from: [ ]] 
1619625083783 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083789 [Data from: [w]] 
1619625083789 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083822 [Data from: [ ]] 
1619625083822 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083831 [Data from: [w]] 
1619625083832 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083859 [Data from: [ ]] 
1619625083860 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083885 [Data from: [w]] 
1619625083887 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083898 [Data from: [ ]] 
1619625083899 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083911 [Data from: [w]] 
1619625083912 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083936 [Data from: [ ]] 
1619625083936 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083953 [Data from: [w]] 
1619625083953 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083973 [Data from: [ ]] 
1619625083973 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083981 [Data from: [w]] 
1619625083981 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625083995 [Data from: [ ]] 
1619625083995 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625084630 [Data from: [w]] 
1619625084630 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 98, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625084751 [Data from: [ ]] 
1619625084751 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625084911 [Data from: [d]] 
1619625084911 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 87, in handle_command
    self.set_speed([self._current_speed,self._current_speed*0.5])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625085049 [Data from: [ ]] 
1619625085049 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625086050 [Data from: [a]] 
1619625086051 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 80, in handle_command
    self.set_speed([self._current_speed*0.5,self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625086200 [Data from: [ ]] 
1619625086200 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625086485 [Data from: [d]] 
1619625086486 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 87, in handle_command
    self.set_speed([self._current_speed,self._current_speed*0.5])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 102, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625086582 [Data from: [ ]] 
1619625086582 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
1619625086621 [Data from: [s]] 
1619625086622 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 105, in handle_command
    self.set_speed([-self._current_speed, -self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 101, in set_speed
    command += struct.pack("<H", speed)
error: ushort format requires 0 <= number <= USHRT_MAX
] 
1619625086866 [Data from: [ ]] 
1619625086866 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 107, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 63, in write_command
    rospy.loginfo('{0}'.format(command_send))
NameError: global name 'rospy' is not defined
] 
